src/include/portability.h

```diff
@@ -52,4 +52,19 @@
 #define dlerror_reset() dlerror()
 #define SHAREDLIB_EXT "so"
 
+#ifdef PBS_MOM
+#define TRAILING_CHAR '/'
+#define verify_dir(dir_val, isdir, sticky, disallow, fullpath) tmp_file_sec(dir_val, isdir, sticky, disallow, fullpath)
+#define STAT_CHKPT_FILE(file_name, buf) stat(file_name, buf)
+#define FULLPATH 1
+#define PLATFORM_SPECIFIC_USAGE "[-S port]\n"
```

Comment:
let's not have this, instead make two functions for usage() one for linux and other for windows